apps/nsmgr/nsmgr.yaml

```diff
@@ -51,7 +51,7 @@ spec:
           resources:
             limits:
               memory: 100Mi
-              cpu: 200m
+              cpu: 400m
```

Comment:
Could you please look at https://github.com/networkservicemesh/deployments-k8s/pull/3050?

Reply:
Looks like we need either to merge #3050 first, or just replace it with this PR.
Updated this PR.